packages/wouter/test/memory-location.test.ts

```diff
@@ -33,6 +44,16 @@ it('should return location hook that has initial path "/" by default', () => {
   unmount();
 });
 
+it('should return search hook that has initial query "" by default', () => {
+  const { searchHook } = memoryLocation();
+
+  const { result, unmount } = renderHook(() => searchHook());
+  const [value] = result.current;
+
+  expect(value).toBe("");
+  unmount();
+});
+
```

Comment:
One test case that is missing is when `memoryLocation` accepts `searchPath` as a parameter.